### PR TITLE
Add TinkerHost.net

### PR DESCRIPTION
**Pull request by greenreader9**
TinkerHost is looking to use DomainConnect to easily allow users to configure required DNS entries for TinkerHost Hosting with their provider.

We are looking to integrate primarily with Cloudflare.

Please let me know if any additional tests or adjustments need to be made.

EDIT ([e851306](https://github.com/Domain-Connect/Templates/pull/549/commits/4acb53bce074e99f5286b533b0b8353ac02ada4a) & [4acb53b](https://github.com/Domain-Connect/Templates/pull/549/commits/e8513067dafa82cc925fec77ed8631541071f9ed)): Renamed the file to not include an uppercase letter

### CODE REVIEW

File: tinkerhost.net.th-customdomain.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "providerId": "tinkerhost.net",
+  "providerName": "TinkerHost",
+  "serviceId": "th-customDomain",
+  "serviceName": "TinkerHost Hosting",
+  "version": 1,
+  "syncPubKeyDomain": "domainConnect.tinkerhost.net",
```
Comment (pawel-kow) [inline tinkerhost.net.th-customdomain.json:7]: it is not an errror, but out of purity please just use low caps on host names

**Comment by pawel-kow** [comment]
For cloudflare I'd recommend running the [linter](https://github.com/Domain-Connect/dc-template-linter) with cloudflare profile to check compatibility with this provider & clean up the template:
`dc-template-linter -increment -inplace <template-file>`

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: tinkerhost.net.th-customdomain.json (hunk 1 above)
Comment (greenreader9) [reply tinkerhost.net.th-customdomain.json]: Not a problem, updated to resolve

**Comment by greenreader9** [comment]
Resolved with [c33054d](https://github.com/Domain-Connect/Templates/pull/549/commits/c33054d5587b3289f5a241659a0f38cd5f43d0e9). Please review @pawel-kow for merge.

**Pull request merged**